### PR TITLE
Update install script (removing bug...)

### PR DESCRIPTION
**Pull request by pi-dev500**

### CODE REVIEW

File: atlauncher/install [hunk 1]
```diff
@@ -1,6 +1,4 @@
 #/usr/bin/bash
-if [ ! -f SetupATpi.sh ];then
 wget -nv https://raw.githubusercontent.com/pi-dev500/MinecraftMicrosoftPILauncher/main/setupATpi.sh || exit 1
-fi
-./SetupATpi.sh || exit 1
```
Comment (Itai-Nelken) [inline atlauncher/install:5]: why did you remove this line and the one bellow? now it only downloads the script and doesn't execute it if I understand it correctly.

**Comment by pi-dev500** [comment]
@Itai-Nelken , can you now test the script ?
Normaly It works.
The command is the one of my readme.md

**Comment by Itai-Nelken** [comment]
> @Itai-Nelken , can you now test the script ?
> Normaly It works.
> The command is the one of my readme.md

I don't have time today, but I'll test it tommorow.

**Comment by Itai-Nelken** [comment]
@pi-dev500:
```bash
pi@raspberrypi:~ $ wget -qO- https://github.com/chunky-milk/pi-bashscripts-files/raw/main/atlauncher/install | bash
2021-02-05 15:50:52 URL:https://raw.githubusercontent.com/pi-dev500/MinecraftMicrosoftPILauncher/main/setupATpi.sh [12040/12040] -> "setupATpi.sh" [1]
bash: line 5: ./SetupATpi.sh: No such file or directory
pi@raspberrypi:~ $ 
```

**Comment by Itai-Nelken** [comment]
the script was downloaded:
https://i.imgur.com/wNDsE0s.png

**Comment by pi-dev500** [comment]
This works for me

screenshotat

**Comment by pi-dev500** [comment]
> the script was downloaded:
> https://i.imgur.com/wNDsE0s.png

That's just because the readme mention how to install with the old script. This command will work if chunky-milk merge the pull request.    For the instant, we can just download the script of my pull request and execute It! Or we can also use the pi-apps package of my repository.

**Comment by Itai-Nelken** [comment]
@pi-dev500 your script works!
https://i.imgur.com/ONkUtXg.png

**Comment by Itai-Nelken** [comment]
https://user-images.githubusercontent.com/70802936/107070053-71d9b280-67eb-11eb-8504-bee92e783725.mp4

**Comment by ryanfortner** [comment]
cool! should i merge now
